tests/v2/integration/rbac/rbac_test.go

```diff
@@ -0,0 +1,412 @@
+package rbac
```

Comment:
So with this PR, we are moving rbac tests to integration as well?

Reply:
yes that is correct. We separated additional tests from the rbac_test.go and moving the P0 rbac checks to the integration suite.